SQL/0000-00-00-schema.sql

```diff
@@ -2012,3 +2016,23 @@ CREATE TABLE `feedback_mri_comments` (
   CONSTRAINT `FK_feedback_mri_comments_2` FOREIGN KEY (`PredefinedCommentID`) REFERENCES `feedback_mri_predefined_comments` (`PredefinedCommentID`),
   CONSTRAINT `FK_feedback_mri_comments_3` FOREIGN KEY (`FileID`) REFERENCES `files` (`FileID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `consent` (
```

Comment:
I would call this table `consent_types` to be more descriptive. (I'm also not sure there's much point in having default values for non-nullable unique keys, but that doesn't really matter..)